test/tall/function/default_value.unit

```diff
@@ -66,9 +66,10 @@ f([
 f([callback(SomeType parameter) = const CallableClass(someLongConstantArgument, anotherConstantArgument)]) {}
 <<<
 f([
-  callback(SomeType parameter) =
-      const CallableClass(
-        someLongConstantArgument,
-        anotherConstantArgument,
-      ),
+  callback(
+    SomeType parameter,
+  ) = const CallableClass(
+    someLongConstantArgument,
+    anotherConstantArgument,
+  ),
```

Comment:
Is this part of the change reflected in the commit message?

I'm having a hard time understanding which part of the code change caused this.

Reply:
works for me